core/modules/wsched/BlendScheduler.cc

```diff
@@ -137,6 +137,134 @@ void BlendScheduler::_sortScanSchedulers() {
 }
 
 
+void BlendScheduler::queCmd(util::Command::Ptr const& cmd) {
+    std::vector<util::Command::Ptr> vect;
+    vect.push_back(cmd);
+    queCmd(vect);
+}
+
+
+/// &&& Change this to accept a vector of util::Command::Ptr const&
+void BlendScheduler::queCmd(std::vector<util::Command::Ptr> const& cmds) {
+    // Do the book keeping and determine which queue the tasks in 'cmds'
+    // should be added to. All tasks in 'cmds' must belong to the same
+    // user query and will go to the same scheduler.
+    LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd a");
+    bool first = true;
+    SchedulerBase::Ptr s = nullptr;
+    bool onInteractive = false;
+    for (auto const& cmd : cmds) {
+        LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd b");
+        wbase::Task::Ptr task = dynamic_pointer_cast<wbase::Task>(cmd);
+        if (task == nullptr) {
+            LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd c");
+            // These should be few and far between.
+            LOGS(_log, LOG_LVL_INFO, "BlendScheduler::queCmd got control command");
+            if (cmds.size() > 1) {
+                throw Bug("cmds.size() > 1 when no task was set.");
+            }
+            {
+                util::LockGuardTimed guard(util::CommandQueue::_mx, "BlendScheduler::queCmd a");
+                _ctrlCmdQueue.queCmd(cmd);
+            }
+            notify(true); // notify all=true
+            LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd c99");
+            return;
+        }
+
+        if (first) {
+            QSERV_LOGCONTEXT_QUERY_JOB(task->getQueryId(), task->getJobId());
+        }
+        LOGS(_log, LOG_LVL_WARN, "&&& BlendScheduler::queCmd d");
```

Comment:
These single letter messages should be posted at the level `WARN` since they don't contain anything resembling warnings. I think the right level should be `DEBUG` or `TRACE`.

Reply:
This must be from and intermediate commit as it has already been removed.